Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -343,7 +350,19 @@ void SVertexer::buildT2V(const o2::globaltracking::RecoContainer& recoData) // a
         }
       }
       const auto& trc = recoData.getTrackParam(tvid);
-      if (!acceptTrack(tvid, trc)) {
+
+      bool ionisingTrack = false;
+      auto tpcGID = recoData.getTPCContributorGID(tvid);
+      if (tpcGID.isIndexSet()) {
+        auto& tpcTrack = recoData.getTPCTrack(tpcGID);
+        float dEdxTPC = tpcTrack.getdEdx().dEdxTotTPC;
+        if (dEdxTPC > mSVParams->minTPCdEdx && trc.getP() > mSVParams->minMomTPCdEdx) // accept high dEdx tracks (He3, He4)
+        {
+          ionisingTrack = true;
+        }
+      }
+
+      if (!acceptTrack(tvid, trc) && !ionisingTrack) {
```

Comment:
Instead of disabling the dca cut, should not we in general pass the info about PID to the acceptTrack so that it is accounted in the (propagation for the ) DCA calculation?

Reply:
The issue is not due to the wrong PID employed for computing the DCA; it is more the fact that He3 and He4 daughters point into the direction of the mother as they inherit most of the initial momentum. For these particles the DCA cut should be removed/released

Reply:
OK, understood, looks fine for me, I think you can remove the draft tag to trigger CI.

Reply:
Done! Thanks